avocado/core/result.py

```diff
@@ -165,6 +166,7 @@ def end_test(self, state):
         """
         self.tests_run += 1
```

Comment:
OK, could you please add `#FIXME:` or `#TODO` mark that it is going to be removed?